tests/chainer_tests/functions_tests/array_tests/test_as_strided.py

```diff
@@ -0,0 +1,305 @@
+import unittest
+
+from chainer import testing, Variable, grad, cuda
+
+import numpy as np
+
+from chainer.functions import as_strided
+from chainer.functions.array.as_strided import _stride_array
+
+
+@testing.parameterize(
+    {'dtype': np.float16},
+    {'dtype': np.float32},
+    {'dtype': np.float64},
+    {'dtype': np.int16},
+    {'dtype': np.int32},
+    {'dtype': np.int64}
+)
+class TestStrideArray(unittest.TestCase):
+    def check_flip(self, xp):
+        x = xp.arange(4, dtype=self.dtype)
+        y = _stride_array(x, (4,), (-1,), 3)  # [3, 2, 1, 0]
+        y_expected = xp.flip(x, axis=0)
```

Comment:
NumPy v1.9.3 does not have `flip`.
```suggestion
        y_expected = x[::-1]
```

Reply:
Thanks. fixed.